types/is-url-superb/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for is-url-superb 2.0
+// Project: https://github.com/sindresorhus/is-url-superb
+// Definitions by: Michael Strobel <https://github.com/kryops>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = isUrl;
+
+declare function isUrl(str: string): boolean;
+declare namespace isUrl {}
```

Comment:
This is https://github.com/Microsoft/TypeScript/issues/5073 all over again.
Has there a good solution been found yet?
* `import isUrl = require('is-url-superb');` does not work with `"module": "ES2015"`
* `import isUrl from 'is-url-superb'` + `"allowSyntheticDefaultExports": true` works with webpack/babel, but breaks CommonJS (e.g. for testing)
* `import isUrl = require('is-url-superb');` with `"module": "esnext"` does not error, but does not seem to emit anything